### PR TITLE
feat: Updated new meeting settings area

### LINKED ISSUE #5603: Updated new meeting settings area
See #5589 for context
settings area

## Acceptance criteria
* [ ] adapt settings area to new design, this includes
  * [ ] icebreaker checkbox
  * [ ] template select button
  * [ ] team select dropdown, **not** the team member grid

### PR DESCRIPTION
**Pull request by igorlesnenko**
Fixes #5603

**Next PR: #7001**

image

image

How to test:

- Open new meeting dialog
- Make sure all new controls works for all meeting types bot on mobile and desktop

### CODE REVIEW

**Comment by igorlesnenko** [comment]
@JimmyLv can you review this PR please?

File: packages/client/components/NewMeeting.tsx [hunk 1]
```diff
@@ -33,27 +31,42 @@ interface Props {
...
+const NewMeetingDialog = styled(DialogContainer)({
   width: '800px',
-  borderRadius: isDesktop ? Radius.FIELD : 0,
-  minWidth: isDesktop ? 'unset' : '100vw',
-  maxHeight: isDesktop ? 'unset' : '100vh',
-  minHeight: isDesktop ? 'unset' : '100vh',
-}))
+  borderRadius: Radius.FIELD,
+
+  [MEDIA_QUERY_FUZZY_TABLET]: {
```
Comment (JimmyLv) [inline packages/client/components/NewMeeting.tsx:63]: +2 #TIL I like the pattern of handling responsive styles this way, rather than by `isDesktop`

**Review by JimmyLv — APPROVED** [review]
LGTM! Great to see we have a prettier new meeting settings area 👍

**Comment by mattkrick** [comment]
+1 For mobile, can we center the active meeting in the carousel so the previous & next options are both slightly visible? that way folks can go back & forth just by tapping instead of swiping

**Review by mattkrick — CHANGES_REQUESTED** [review]
see comment on centering active meeting

**Comment by igorlesnenko** [comment]
@mattkrick I tried to fix that quickly, but was unable to, as possibly, there is a bug in the library.

Also, since in this PR I only touched settings part of the dialog, I created a separate issue and will look into it: #7040

Is it ok?

**Comment by mattkrick** [comment]
@igorlesnenko i think i got it working: https://www.loom.com/share/d9a85b1904be463ea3cb7401e8f4e47d

If you like what you see, go ahead & merge https://github.com/ParabolInc/parabol/pull/7061 into this PR.

aside from that, lgtm

**Comment by igorlesnenko** [comment]
@mattkrick I left comments in your PR: https://github.com/ParabolInc/parabol/pull/7061#pullrequestreview-1071109618

I'm merging this PR for now

**Pull request merged**